be/src/vec/columns/column_object.cpp

```diff
@@ -1513,4 +1514,78 @@ Status ColumnObject::sanitize() const {
     return Status::OK();
 }
 
+void ColumnObject::Subcolumn::replace(Field field, int row_num) {
+    finalize();
+    Field new_field;
+    if (!field.is_null()) {
+        convert_field_to_type(field, *least_common_type.get(), &new_field);
+        data.back()->replace(new_field, row_num);
+    } else {
+        data.back()->replace(Null(), row_num);
+    }
+}
+
+void ColumnObject::replace_column_data(const IColumn& col, size_t row, size_t self_row) {
+    DCHECK(size() > self_row);
+    DCHECK(col.size() > row);
```

Comment:
here may be should clear first when self_row == 0?

Reply:
no variant may contains none vairble length column